services/device-registry-base/src/main/java/org/eclipse/hono/service/management/device/DeviceAndGatewayAutoProvisioner.java

```diff
@@ -223,17 +226,7 @@ private static Device createDeviceInformation(final boolean isGateway, final Str
         return device;
     }
 
-    /**
-     * Checks if auto-provisioning is enabled from the tenant's trusted CA entry which has been
-     * used to authenticate the device.
-     *
-     * @param tenant The tenant information to check if auto-provisioning is enabled or not.
-     * @param certificate The client certificate that devices used for authentication.
-     *                    If the certificate is {@code null} then {@code false} is returned.
-     * @param span The active OpenTracing span for this operation.
-     * @return {@code true} if auto-provisioning is enabled.
-     */
-    private static boolean isAutoProvisioningEnabledForTenant(final Tenant tenant, final X509Certificate certificate,
+    private boolean isAutoProvisioningEnabledForTenant(final Tenant tenant, final X509Certificate certificate,
```

Comment:
given that the check is scoped to a tenant, it seems reasonable to include the tenant in the message being logged via the logging framework, doesn't it?